src/ray/common/ray_syncer/ray_syncer.cc

```diff
@@ -355,6 +355,10 @@ bool RaySyncer::OnDemandBroadcasting(MessageType message_type) {
   return false;
 }
 
+void RaySyncer::OnDemandBroadcasting(const RaySyncMessage &message) {
+  BroadcastMessage(std::make_shared<const RaySyncMessage>(std::move(message)));
```

Comment:
The move operations will not affect variables with `const &`

Reply:
Will do.